src/gc/gcinterface.h

```diff
@@ -546,16 +546,16 @@ class IGCHeap {
     virtual HRESULT Initialize() = 0;
 
     // Returns whether nor this GC was promoted by the last GC.
-    virtual BOOL IsPromoted(Object* object) = 0;
+    virtual bool IsPromoted(Object* object) = 0;
 
     // Returns true if this pointer points into a GC heap, false otherwise.
-    virtual BOOL IsHeapPointer(void* object, BOOL small_heap_only = FALSE) = 0;
+    virtual bool IsHeapPointer(void* object, bool small_heap_only = false) = 0;
```

Comment:
yeah, I was a bit surprised too... perhaps we are disabling this warning somewhere (that would be surprising too).